autoload/lsp/ui/vim/output.vim

```diff
@@ -156,7 +156,10 @@ function! s:setcontent(lines, ft) abort
     endif
 
     setlocal readonly nomodifiable
-    let &l:filetype = a:ft . '.lsp-hover'
+    try
```

Comment:
You could also consider

`silent! let &l:filetype = a:ft . '.lsp-hover'`

Reply:
Thanks, I like that much more. Changed in #446.